roles/wordpress-setup/tasks/database.yml

```diff
@@ -11,6 +11,7 @@
 - name: Create/assign database user to db and grant permissions
   mysql_user: name="{{ item.value.env.db_user }}"
               password="{{ item.value.env.db_password }}"
+              append_privs=true
```

Comment:
New commit coming within minutes. I was not sure, as if you see the same page I pasted above, the example has `true`. :confused: